source/addComment.js

```diff
@@ -1,3 +1,39 @@
 $(document).ready(function(){
-  // add your code here
-})
+
+  $(document.body).append(
+   $('<form/>').prop('id', 'new_comment').append(
+     $("<textarea/>", {
+       type: 'text',
+       id: 'textArea',
+       placeholder: 'Your comment here...'
+       }),
+     $("<br/>"),
+     $("<input/>", {
+       type: 'text',
+       id: 'authorName',
+       placeholder: 'Author Name'
+       }),
+       $("<input/>", {
+         type: 'submit',
+         id: 'submit',
+         value: 'Submit'
+       })
+   )
+  )
+
+   $('#new_comment').submit(function() {
+    var comment = $('#textArea').val()
+    var author = $('#authorName').val()
+    $("#comment_list").append('<li>' + comment + '<br>' + '<span class="author">' + author + '</span>' + '</li>')
```

Comment:
could we extract this out into it's own function so we could maybe reuse it in the future?